src/5e-SRD-Rules-Sections.json

```diff
@@ -0,0 +1,179 @@
urn, you can **move** a distance up to your speed and **take one action**. You decide whether to move first or take your action first. Your speed— sometimes called your walking speed—is noted on your character sheet.\n\nThe most common actions you can take are described in the \"Actions in Combat\" section later in this chapter. Many class features and other abilities provide additional options for your action.\n\nThe \"Movement and Position\" section later in this chapter gives the rules for your move.\n\nYou can forgo moving, taking an action, or doing anything at all on your turn. If you can’t decide what to do on your turn, consider taking the Dodge or Ready action, as described in \"Actions in Combat.\"\n\n#### Bonus Actions\n\nVarious class features, spells, and other abilities let you take an additional action on your turn called a bonus action. The Cunning Action feature, for example, allows a rogue to take a bonus action. You can take a bonus action only when a special ability, spell, or other feature of the game states that you can do something as a bonus action. You otherwise don’t have a bonus action to take.\n\nYou can take only one bonus action on your turn, so you must choose which bonus action to use when you have more than one available.\n\nYou choose when to take a bonus action during your turn, unless the bonus action’s timing is specified, and anything that deprives you of your ability to take actions also prevents you from taking a bonus action.\n\n#### Other Activity on Your Turn\n\nYour turn can include a variety of flourishes that require neither your action nor your move.\n\nYou can communicate however you are able, through brief utterances and gestures, as you take your turn.\n\nYou can also interact with one object or feature of the environment for free, during either your move or your action. For example, you could open a door during your move as you stride toward a foe, or you could draw your weapon as part of the same action you use to attack.\n\nIf you want to interact with a second object, you need to use your action. Some magic items and other special objects always require an action to use, as stated in their descriptions.\n\nThe GM might require you to use an action for any of these activities when it needs special care or when it presents an unusual obstacle. For instance, the GM could reasonably expect you to use an action to open a stuck door or turn a crank to lower a drawbridge.\n\n### Reactions\n\nCertain special abilities, spells, and situations allow you to take a special action called a reaction. A reaction is an instant response to a trigger of some kind, which can occur on your turn or on someone else’s. The opportunity attack, described later in this chapter, is the most common type of reaction.\n\nWhen you take a reaction, you can’t take another one until the start of your next turn. If the reaction interrupts another creature’s turn, that creature can continue its turn right after the reaction.\n",
```

Comment:
Sorry. I meant to chime in about this, but instead of doing:
```
The Order of Combat
-------------------
```
or:
```
Appendix
========
```
can we make use of [markdown headers](https://www.markdownguide.org/basic-syntax/#headings):
```
## The Order of Combat
```
and
```
# Appendix
```

Reply:
Sure yeah no problem